articles/api-auth/using-rules/password.md

```diff
@@ -0,0 +1,114 @@
+---
+description: How to add a webtask and run it each time a Resource Owner Password grant is executed.
+---
+
+# Using Rules with Resource Owner Password Grant
+
+You can write functions in JavaScript and have them run every time a [Resource Owner Password grant](/api-auth/grant/password) is executed, using [webtasks](https://webtask.io/).
+
+## Before you begin
+
+<%= include('./_includes/_prerequisites.md') %>
```

Comment:
It isn't :) It's required for `client_credentials` and this bullet was left there after the merging of these two docs. I'll fix that right away. Thanks! 👍